backend/package.json

```diff
@@ -21,7 +21,7 @@
     "prisma": "^5.13.0"
   },
   "devDependencies": {
-    "@cloudflare/workers-types": "^4.20240403.0",
+    "@cloudflare/workers-types": "^4.20240512.0",
```

Comment:
Please state the reasoning behind this change. 
If it was unintentional, rollback to previous state.